modules/nw-dpu-running-workloads.adoc

```diff
@@ -48,7 +48,9 @@ spec:
         openshift.io/dpu: '1'
 ----
 +
-<1> The name of the node where the workload is deployed.
+* `spec.nodeSelector`: The node selector schedules the pod on the node with the DPU resource. You have two primary methods for this:
+** Specific node scheduling: You can use a selector like `kubernetes.io/hostname: <node_name>` to schedule the pod onto a specific named node as shown in the example YAML.
+** Flexible DPU host scheduling: Alternatively, the label `dpu.config.openshift.io/dpuside: "dpu-host"` is created by the DPU Operator and can be used to schedule the workload on any host with any DPU. The workload  joins that DPU secondary network. When the label on the node is `dpu.config.openshift.io/dpuside: "dpu"`, this signifies that the node is a DPU.
```

Comment:
It feels like this is redundant. Better to say that we're using standard k8s primitives so what you learned from the basic k8s manual applies to DPUs too. Maybe go through the demo to get the feel for this:

https://www.youtube.com/watch?v=SUniRV2Bzr4 starting at 16 minutes

Reply:
Propose rewrite:
```
* `spec.nodeSelector`: The node selector schedules the pod on the node with the DPU resource. You can use any selector for this
** Hint: The label `dpu.config.openshift.io/dpuside: "dpu-host"` is created by the DPU Operator and can be used to schedule the workload on any host with any DPU. The workload  automatically joins that DPU secondary network. When the label on the node is `dpu.config.openshift.io/dpuside: "dpu"`, this signifies that the node is a DPU. The label `dpu.config.openshift.io/dpuside` is managed by the DPU Operator.
```

Reply:
@kquinn1204 I leave the phrasing up to you. Can you make sure that the intent of that message isn't lost?